.github/workflows/deploy.yml

```diff
@@ -25,6 +25,12 @@ jobs:
         # Docker is preinstalled.
         run: nix build .#dockerImage && ./result | docker load
 
+      - name: Assume AWS role
+        uses: aws-actions/configure-aws-credentials@v4
```

Comment:
Just commenting early to say that I've been trying to avoid unnecessary use of actions as they're proprietary to GitHub, making the code less portable to other CI systems. They often do little more than a line or two of shell.

Reply:
Thanks for the feedback. Even though I do not agree I will take it in consideration.